docs_src/pipelinebenchmarking.md

```diff
@@ -62,6 +62,7 @@ Run benchmarking pipelines:
 ```bash
 sudo ./benchmark.sh --pipelines <number of pipelines> --logdir <output dir>/data --init_duration 30 --duration 120 --platform <core|xeon|dgpu.x> --inputsrc <ex:4k rtsp stream with 10 objects>
 ```
+Note: you need to run `./dokcer-build-igt.sh` under /vision-self-checkout folder before you run above benchmarking pipeline for Core
```

Comment:
Agree, this is redundant as well as the very beginning of this page already states 

Pre-rerequisites:
Pipeline setup needs to be done first, pipeline setup documentation be found [HERE](https://github.com/intel-retail/vision-self-checkout/blob/f3466a01adce4d079bc62e68448a6b3c7b077673/docs_src/pipelinesetup.md)

and in the pipelinesetup.md it has the details for docker build igt already. This would reduce maintaining the same requirement in multiple places. Thoughts?

Reply:
i agree